docs/dev/table/sourceSinks.md

```diff
@@ -743,9 +744,25 @@ public class MySystemConnector extends ConnectorDescriptor {
   }
 }
 {% endhighlight %}
+
+The descriptor can then be used in the API as follows:
```

Comment:
?
```suggestion
The descriptor can then be used to create a table in the table environment.
```

Reply:
How about "create a table with the table environment"?

Reply:
Sounds good